### ISSUE TITLE
[BUG] Huge lag & error on player join

### ISSUE THREAD

**Issue by SlenkyDev** [ItsWagPvP/EconomyPlus#38]
**Describe the bug**
Using MySQL on 1.18.2, when a player joins, the server TPS goes down to 5 for a couple of seconds and this gets thrown in the console

```
[09:52:34 INFO]: UUID of player Slenk is 
[09:52:44 ERROR]: --- DO NOT REPORT THIS TO PURPUR - THIS IS NOT A BUG OR A CRASH  - git-Purpur-1632 (MC: 1.18.2) ---
[09:52:44 ERROR]: The server has not responded for 10 seconds! Creating thread dump
[09:52:44 ERROR]: ------------------------------
[09:52:44 ERROR]: Server thread dump (Look for plugins here before reporting to Purpur!):
[09:52:44 ERROR]: Ticking entity: minecraft:player, entity class: net.minecraft.server.level.EntityPlayer
[09:52:44 ERROR]: Entity status: removed: false, valid: true, alive: true, is passenger: false
[09:52:44 ERROR]: Entity UUID: 3141c9a9-cf47-3924-a61e-75acdbc83e8e
[09:52:44 ERROR]: Position: world: 'spawn' at location (0.5, 79.0, 0.5)
[09:52:44 ERROR]: Velocity: (0.0, 0.0, 0.0) (in blocks per tick)
[09:52:44 ERROR]: Entity AABB: AABB[0.19999998807907104, 79.0, 0.19999998807907104] -> [0.800000011920929, 80.79999995231628, 0.800000011920929]
[09:52:44 ERROR]: Total packets processed on the main thread for all players: 81317
[09:52:44 ERROR]: ------------------------------
[09:52:44 ERROR]: Current Thread: Server thread
[09:52:44 ERROR]:       PID: 26 | Suspended: false | Native: false | State: WAITING
[09:52:44 ERROR]:       Stack:
[09:52:44 ERROR]:               java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
[09:52:44 ERROR]:               java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
[09:52:44 ERROR]:               java.base@17.0.2/java.util.concurrent.CompletableFuture$Signaller.block(CompletableFuture.java:1864)
[09:52:44 ERROR]:               java.base@17.0.2/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
[09:52:44 ERROR]:               java.base@17.0.2/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
[09:52:44 ERROR]:               java.base@17.0.2/java.util.concurrent.CompletableFuture.waitingGet(CompletableFuture.java:1898)
[09:52:44 ERROR]:               java.base@17.0.2/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2072)
[09:52:44 ERROR]:               EconomyPlus.jar//me.itswagpvp.economyplus.database.mysql.MySQL.getList(MySQL.java:214)
[09:52:44 ERROR]:               EconomyPlus.jar//me.itswagpvp.economyplus.database.misc.DatabaseType$2.getList(DatabaseType.java:86)
[09:52:44 ERROR]:               EconomyPlus.jar//me.itswagpvp.economyplus.events.Join.onPlayerJoin(Join.java:21)
[09:52:44 ERROR]:               com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor176.execute(Unknown Source)
[09:52:44 ERROR]:               org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75)
[09:52:44 ERROR]:               org.bukkit.plugin.EventExecutor$$Lambda$5794/0x0000000801cae1d0.execute(Unknown Source)
[09:52:44 ERROR]:               co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76)
[09:52:44 ERROR]:               org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[09:52:44 ERROR]:               org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:671)
[09:52:44 ERROR]:               net.minecraft.server.players.PlayerList.postChunkLoadJoin(PlayerList.java:363)
[09:52:44 ERROR]:               net.minecraft.server.players.PlayerList.lambda$placeNewPlayer$1(PlayerList.java:305)
[09:52:44 ERROR]:               net.minecraft.server.players.PlayerList$$Lambda$9396/0x0000000802dee368.run(Unknown Source)
[09:52:44 ERROR]:               net.minecraft.server.network.ServerGamePacketListenerImpl.tick(ServerGamePacketListenerImpl.java:320)
[09:52:44 ERROR]:               net.minecraft.network.Connection.tick(Connection.java:576)
[09:52:44 ERROR]:               net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:231)
[09:52:44 ERROR]:               net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1651)
[09:52:44 ERROR]:               net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:522)
[09:52:44 ERROR]:               net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1483)
[09:52:44 ERROR]:               net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1247)
[09:52:44 ERROR]:               net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:321)
[09:52:44 ERROR]:               net.minecraft.server.MinecraftServer$$Lambda$4140/0x00000008014b86a8.run(Unknown Source)
[09:52:44 ERROR]:               java.base@17.0.2/java.lang.Thread.run(Thread.java:833)
[09:52:44 ERROR]: ------------------------------
[09:52:44 ERROR]: --- DO NOT REPORT THIS TO PURPUR - THIS IS NOT A BUG OR A CRASH ---
[09:52:44 ERROR]: ------------------------------
[09:52:47 INFO]: Slenk[<ip address withheld>] logged in with entity id 2903 at ([spawn]0.5, 79.0, 0.5)
```

**To Reproduce**
1. Use MySQL
2. Join the server
3. Enjoy the lag and error in console

**Comment by ItsWagPvP** [comment]
can you send a timings paste or a spark? thanks

**Comment by SlenkyDev** [comment]
Sorry, don't use the plugin anymore.
------- Original Message -------

**Comment by SlenkyDev** [comment]
? Did you fix it or something

**Comment by ItsWagPvP** [comment]
fixed it in 4.4
